picard/formats/asf.py

```diff
@@ -221,10 +220,7 @@ def _remove_deleted_tags(self, metadata, tags):
         for tag in metadata.deleted_tags:
             real_name = self._get_tag_name(tag)
             if real_name and real_name in tags:
-                if tag == 'totaldiscs':
-                    tags[real_name] = metadata['discnumber']
```

Comment:
Something changed in the way mutagen behaves with the WM/PartOfSet tag, one is useless without the other. (disc number and total discs)